packages/@aws-cdk/aws-amplify/package.json

```diff
@@ -93,6 +93,7 @@
   "dependencies": {
     "@aws-cdk/aws-codebuild": "0.0.0",
     "@aws-cdk/aws-codecommit": "0.0.0",
+    "@aws-cdk/aws-ec2": "0.0.0",
```

Comment:
This is probably the source of the issue (and where you have it added below as well). Was adding this here a mistake? I don't see any other changes to `aws-amplify` so the dependency addition here doesn't make a lot of sense to me.